src/Symfony/Bundle/FrameworkBundle/Test/LoggerAssertionsTrait.php

```diff
@@ -40,9 +44,21 @@ public static function assertLogMatches(string $expectedRegex, string $level = L
 
     public static function assertLogContains(string $expectedLog, string $level = Logger::DEBUG): void
     {
+        self::ensureMonologHandlerIsAvailable();
+
         /** @var TestHandler $logger */
         $logger = self::getContainer()->get('monolog.handler.test');
 
         self::assertTrue($logger->hasRecordThatContains($expectedLog, $level));
     }
+
+    /**
+     * @internal
+     */
+    private static function ensureMonologHandlerIsAvailable(): void
+    {
+        if (!self::getContainer()->has('monolog.handler.test')) {
+            self::fail('The "monolog.handler.test" service is not available. Try registering the service "Monolog\Handler\TestHandler" as "monolog.handler.test" in your test configuration.');
```

Comment:
What about mentioning using the Monolog bundle configuration if available?

```yml
when@test:
    monolog:
        handlers:
            test: { type: test }
            main:
                type: fingers_crossed
                # […]
```

This would work, because it'll be registered through the Monolog bundle extension in the handlers chain, but I don't get how just registering a `monolog.handler.test` would ensure this logger handler is used when using the logger in your application.

Reply:
You are right, I should belongs to the Monolog bridge, as it only works with Monolog. Or I have to reimplement `hasRecord*` methods for generic loggers (which I do not want to do :smile:).

I'm going to move it, thanks for your feedback

Reply:
I don't well know this part, especially how to use this bridge in tests. Would you help me?